src/scripts/poll.ts

```diff
@@ -1,6 +1,12 @@
 import { checkForErroredJobs } from '../lib/check-jobs'
 import { runStudies } from '../lib/run-studies'
 
+const pollStudiesInterval = parseInt(process.env.POLL_STUDIES_INTERVAL_MINUTES || '2')
```

Comment:
Can we make this default `10` perhaps? Given this is not the long pole in a test cycle loop (the BMA UI steps and container image build in particular are), I'd lean towards something larger just to prevent faster crash loops if we hit weird states.